### ISSUE TITLE
Folder `C:\temp` is not Checked For

### ISSUE THREAD

**Issue by toydotgame** [Redo122/Big-Text-Generator#1]
I will have to manually create the directory `C:\temp` in order for the text file to be created. If the `temp\` directory does not exist, a stack trace is printed:

### Stack Trace
```
Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path 'c:\temp\BigText_Output.txt'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   at Big_Text_Generator.Program.Main(String[] args)
```

**Comment by thardwardy** [comment]
ah, that would be a problem. I'll try and get it fixed later this afternoon if I can.

**Comment by thardwardy** [comment]
Now fixed in v1.1
